src/ngraph/runtime/gpu/gpu_call_frame.cpp

```diff
@@ -0,0 +1,107 @@
+//*****************************************************************************
```

Comment:
Can FunctionInstance be used as call frame or are they really two different things?